src/main/java/org/opentripplanner/netex/mapping/StationMapper.java

```diff
@@ -65,9 +68,33 @@ Station map(StopPlace stopPlace) {
       );
     }
 
+    /* handle IFOTP id stored in KV GlobalID in the DELFI feed to allow matching to OSM */
+    if (stopPlace.getKeyList() != null) {
+      String globalId = stopPlace
+        .getKeyList()
+        .getKeyValue()
+        .stream()
+        .filter(keyValueStructure -> keyValueStructure.getKey().equals("GlobalID"))
```

Comment:
I think this is ok, but we should make the key configurable in the NeTEx configuration